nmdc_api_utilities/test/test_nmdcsearch.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import logging
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+ENV = os.getenv("ENV")
+logging.basicConfig(level=logging.DEBUG)
+from nmdc_api_utilities.nmdc_search import NMDCSearch
+
+
+def test_get_records_by_id():
```

Comment:
I don't understand this - how/why are you grabbing 20 records with the same id?  Each record has its own unique id.